### DISCUSSION TITLE
Expected size of video

### DISCUSSION THREAD

**Discussion by alaflaquiere** [hkchengrex/XMem#1] [Q&A] [answered]
Hi,

Your videos looks great, congrats!

I just tried running your Colab demo with (W=540, H=960) video, but I encountered the following error when running. `prediction = processor.step(frame_torch, mask_torch[1:])`:
> RuntimeError: Sizes of tensors must match except in dimension 2. Expected size 480 but got size 960 for tensor number 1 in the list.

Does the model expect a particular video size?
If that's not the case, any idea what the issue might be?

Thanks!

**Comment by hkchengrex** [comment, marked as answer]
Hi, 

It should work with any size (we did most of the experiment in 480p though). Can you check if the frame and the mask have the same size?

**Reply by alaflaquiere** [reply]
Actually, going back to it today, I realized that the video loaded in the `mp4` variable is not the one fed to the network. Instead, it is loaded again in the `cap` variable, which I hadn't changed.
So totally my fault! It's working now. :)

Thank for the quick reply.

**Reply by hkchengrex** [reply]
Ah, my bad. I literally copy-pasted that visualization code from stackoverflow without realizing that it creates the `mp4` variable. I have made that inline to prevent future confusion. Thanks!
